app/components/back-office-app-services/variables.tf

```diff
@@ -167,3 +172,8 @@ variable "tags" {
   description = "The tags applied to all resources"
   type        = map(string)
 }
+
+variable "resource_group_id" {
```

Comment:
This variable could be removed if we scope the RBAC permissions to the document store as opposed to the whole resource group.